### ISSUE TITLE
[paraglide-sveltekit] compiler warning: `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead

### ISSUE THREAD

**Issue by siliconcat** [opral/inlang-paraglide-js#259]
Using the latest sveltekit (2.73) and @inlang/paraglide-sveltekit (0.12.0), getting the following compiler warning:

```
10:53:38 AM [vite-plugin-svelte] src/lib/components/ui/button/button.svelte:71:20 `a11y-missing-attribute` is no longer valid — please use `a11y_missing_attribute` instead
```

This is added by paraglide to prevent other issues, so now it looks like underscores must be used.

Thanks

**Comment by samuelstroschein** [comment]
Released in `@inlang/paraglide-sveltekit@0.12.1`
